<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mf1ics50WriteBlock</class>
 <widget class="QDialog" name="mf1ics50WriteBlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>486</width>
    <height>186</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>486</width>
    <height>186</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QLineEdit" name="lineEditBlock0">
     <property name="text">
      <string>FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="checkBoxBlock0">
     <property name="text">
      <string>Block 0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLineEdit" name="lineEditBlock1">
     <property name="text">
      <string>FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="checkBoxBlock2">
     <property name="text">
      <string>Block 1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLineEdit" name="lineEditBlock2">
     <property name="text">
      <string>FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QCheckBox" name="checkBoxBlock1">
     <property name="text">
      <string>Block 2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEditBlock3KeyA">
     <property name="text">
      <string>FFFFFFFFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditBlock3Control">
     <property name="text">
      <string>FF078069</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEditBlock3KeyB">
     <property name="text">
      <string>FFFFFFFFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QCheckBox" name="checkBoxBlock3">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Block 3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="radioButtonKeyA">
     <property name="text">
      <string>KEYA</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupKeySelect</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditKeyA">
     <property name="text">
      <string>FFFFFFFFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QRadioButton" name="radioButtonKeyB">
     <property name="text">
      <string>KEYB</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupKeySelect</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLineEdit" name="lineEditKeyB">
     <property name="text">
      <string>FFFFFFFFFFFF</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QCheckBox" name="checkBoxKeyHide">
     <property name="text">
      <string>Hide</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QComboBox" name="comboBoxSector">
     <item>
      <property name="text">
       <string>Sector 0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sector 15</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QPushButton" name="pushButtonReadBlock">
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="pushButtonWriteBlock">
     <property name="text">
      <string>Write</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupKeySelect"/>
 </buttongroups>
</ui>
